pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -65,6 +65,11 @@ private ElasticSearchParticipantDto(ElasticSearchParticipantDto.Builder builder)
 
     protected ElasticSearchParticipantDto() {  }
 
+    public static Optional<ElasticSearchParticipantDto> fromSourceMap(Map<String, Object> sourceMap,
```

Comment:
This is the right place to have this code. I moved it while doing some Elastic deserialization debugging.